lib/internal/worker.js

```diff
@@ -47,6 +47,15 @@ const kIncrementsPortRef = Symbol('kIncrementsPortRef');
 
 const debug = util.debuglog('worker');
 
+const messageTypes = {
+  UP_AND_RUNNING: 'kUpAndRunning',
+  COULD_NOT_SERIALIZ_EERROR: 'kCouldNotSerializeError',
+  ERROR_MESSAGE: 'kErrorMessage',
+  STDIO_PAYLOAD: 'kStdioPayload',
+  STDIO_WANTS_MORE_DATA: 'kStdioWantsMoreData',
+  LOAD_SCRIPT: 'kLoadScript'
+};
```

Comment:
What if we use variables instead of properties of an object? This way, mistakes/typos can be caught at linting time, and errors like the one above `COULD_NOT_SERIALIZ_EERROR` would be more difficult to make (one has to make it at least twice for it to pass linting).

Reply:
Using variables may not prevent typos like `COULD_NOT_SERIALIZ_EERROR` because most of developers are using auto complete when coding. Using properties of an object can make developer benefits from the auto complete (just type `messageTypes.` and these constants will be showed)